src/meter/Meter.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import FlexView from '../flex/FlexView';
+import find from 'lodash/collection/find';
+import { warn } from '../utils/log';
+
+const Range = t.refinement(t.struct({
+  startValue: t.maybe(t.Number),
```

Comment:
@giogonzo i choose to make it implicit cause in most of the cases steps are contiguous, so they can be pre-computed. If the user want to force a new `startValue` it can be possibile as well.